packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -980,16 +874,12 @@ contract('FederatedAttestations', (accounts: string[]) => {
         sig.r,
         sig.s
       )
-      assertLogMatches2(register.logs[0], {
-        event: 'AttestationRegistered',
-        args: {
-          identifier: identifier1,
-          issuer: issuer1,
-          account: account1,
-          issuedOn: nowUnixTime,
-          signer: signer1,
-        },
-      })
+    })
+
+    it('should modify identifierToAddresses and addresstoIdentifiers accordingly', async () => {
```

Comment:
this test case probably doesn't add much once this is covered in the registration cases (see comment above), since you're also already checking that both mappings are modified in sync for each deletion case

Reply:
The registration cases only test the register functionality, not that deletion works correctly

Reply:
Yeah what I mean is slightly different:
- the register tests should assert that after registering an attestation, the attestation has been added to storage for the two arrays; so the first line in this case IMO is covered, you don't need it but not a big deal either

I actually thought there was a separate delete case for a "straightforward deletion" but realizing there isn't a separate one 🤔 In this case, actually fine I think!